src/Microsoft.AspNetCore.Mvc.RazorPages/Internal/IPageExecutionCallbackFeature.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Mvc.RazorPages
+{
+    internal interface IPageExecutionCallbackFeature
```

Comment:
What do you like better about this? I thought we were adding a feature specifically for view data?

Reply:
Ahh, I thought all we settled on was making this a feature.